test/reducers/language_spec.js

```diff
@@ -0,0 +1,36 @@
+import {expect} from 'chai';
+import deepFreeze from 'deep-freeze';
+
+import reducer from '../../src/reducers/language';
+
+describe('language reducer', () => {
+  describe('SET_LANGUAGE', () => {
+    it('sets current language to argument', () => {
+      const initialState = 'sweden';
+      const action = {
+        type: 'SET_LANGUAGE',
+        payload: 'norway'
+      };
+
+      deepFreeze(initialState);
+      deepFreeze(action);
+      const nextState = reducer(initialState, action);
+
+      expect(nextState).to.equal('norway');
+    });
+
+    it('sets current language to null if argument is not spesified or not valid', () => {
```

Comment:
Se `sets current language to null if argument is not specified or not valid`. Returnerer null hvis språk (parameter) ikke er spesifisert eller ikke er godkjent.